docs/guides/tutorials/graphql_apis_with_strawberry.rst

```diff
@@ -378,7 +378,7 @@ This exposes the webserver in port 5000. Now, in your browser, go to
 find that the HTTP basic auth requires us to provide the username and password.
 
 .. image::
-    https://www.edgedb.com/docs/tutorials/strawberry/http_basic.png
+    /docs/tutorials/strawberry/http_basic.png
```

Comment:
When I pull this down and run it locally, these images are not loading for me. They work on the live copy (at docs.edgedb.com), but I'm guessing that's just until the site changes are deployed, which might remove those images at their current locations. Not sure what these paths _should_ be to make them work though. I thought maybe just `/docs` needed to go away, but that didn't seem to work either… 🤔

Reply:
These changes depend on this PR: https://github.com/edgedb/edgedb.com/pull/823. Currently those images are still being loaded from the main site, which will break when the docs-less main site goes live.